include/libcyphal/executor.hpp

```diff
@@ -204,6 +89,27 @@ class IExecutor : public cetl::rtti_helper<IExecutorTypeIdType>
         ///
         using Function = cetl::pmr::function<void(const TimePoint now_time), FunctionMaxSize>;
```

Comment:
It is often convenient, especially for periodic callbacks, to know the scheduled time along with the current time. Would it be difficult to replace this with a struct of two fields: the current time and the scheduled time?

Reply:
OK, in the next pr I'll add new `Callback::Arg` with two fields.